Makefile

```diff
@@ -1,30 +1,30 @@
 .PHONY: build release lint test install uninstall clean reindent
 
-build:
-	jbuilder build @install
-
 release:
-	jbuilder build @install
+	dune build @install
```

Comment:
Do we need ` --profile=release` here?

Reply:
I think we do because the default in Dune is to use the development profile.